TransbankPosSDK/Responses/LastSaleResponse.cs

```diff
@@ -0,0 +1,115 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using Transbank.POS.Utils;
+
+namespace Transbank.POS.Responses
+{
+    public class LastSaleResponse : LoadKeysResponse
+    {
+        private readonly Dictionary<string, int> ParameterMap = new Dictionary<string, int>
+        {
+            { "Ticket", 4},
+            { "AutorizationCode", 5},
+            { "Ammount", 6},
+            { "SharesNumber", 7},
+            { "SharesAmount", 8},
+            { "Last4Digits", 9},
+            { "OperationNumber", 10},
+            { "CardType", 11},
+            { "AccountingDate", 12},
+            { "AccountNumber", 13},
+            { "CardBrand", 14},
+            { "RealDate", 15},
+            { "RealTime", 16},
+            { "EmployeeId", 17},
+            { "Tip", 18}
+        };
+
+        public int Ticket { set; get; }
+        public int AutorizationCode { set; get; }
+        public int Ammount { set; get; }
+        public int SharesNumber { set; get; }
+        public int SharesAmount { set; get; }
+        public int Last4Digits { set; get; }
+        public int OperationNumber { set; get; }
+        public string CardType { set; get; }
+        public DateTime? AccountingDate { set; get; }
+        public long AccountNumber { set; get; }
+        public string CardBrand { set; get; }
+        public DateTime? RealDate { set; get; }
+        public int EmployeeId { set; get; }
+        public int Tip { set; get; }
+
+        public LastSaleResponse(string cresponse) : base(CreateBase(cresponse)) {
+            cresponse = cresponse.Substring(1);
+            string[] fields = cresponse.Split('|');
+
+            Ticket = fields[ParameterMap["Ticket"]].Trim() != "" ? int.Parse(fields[ParameterMap["Ticket"]]) : 0;
```

Comment:
This sounds like something you can refactor like `Ticket = parseIntOrZero(fields[ParameterMap["Ticket"]])` and would be easier to read (of course you would have to implement the helper method `parseIntOrZero` unless there is something in C# that already does that)